Tools/ssl/multissltests.py

```diff
@@ -252,21 +270,29 @@ def _unpack_src(self):
     def _build_src(self):
         """Now build openssl"""
         log.info("Running build in {}".format(self.build_dir))
-        cwd = self.build_dir
-        cmd = ["./config", "shared", "--prefix={}".format(self.install_dir)]
-        cmd.extend(self.compile_args)
+        cmd = [
+            "./config",
+            "shared", "--debug",
+            "--prefix={}".format(self.install_dir)
+        ]
+        env = os.environ.copy()
+        # set rpath
+        env["LD_RUN_PATH"] = self.lib_dir
```

Comment:
Here https://github.com/python/cpython/blob/9fb051f032c36b9f6086b79086b4d6b7755a3d70/Tools/ssl/multissltests.py#L283 is missing which was added on https://github.com/python/cpython/commit/e5f41d2f1e0b8b8e61d5fa427c19bd1ea90fd9a3